specification/communication/data-plane/MediaComposition/readme.md

```diff
@@ -0,0 +1,66 @@
+# communicationservices
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for communicationservices.
+
+## Getting Started
+
+To build the SDKs for My API, simply install AutoRest via `npm` (`npm install -g autorest`) and then run:
+
+> `autorest readme.md`
+
+To see additional help and options, run:
+
+> `autorest --help`
+
+For other options on installation see [Installing AutoRest](https://aka.ms/autorest/install) on the AutoRest github page.
+
+---
+
+## Configuration
+
+### Basic Information
+
+These are the global settings for the communicationservices.
+
+```yaml
+openapi-type: data-plane
+tag: package-2021-12-31-preview
+```
+
+### Tag: package-2021-12-31-preview
+
+These settings apply only when `--tag=package-2021-12-31-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-2021-12-31-preview'
+input-file:
+  - preview/2021-12-31-preview/CommunicationMediaComposition.json
+title:
+  Azure Communication Services
+```
+
+---
+
+# Code Generation
+
+## Swagger to SDK
+
+This section describes what SDK should be generated by the automatic system.
+This is not used by Autorest itself.
+
+## Python
+
+See configuration in [readme.python.md](./readme.python.md)
+
+## Ruby
+
+See configuration in [readme.ruby.md](./readme.ruby.md)
+
+## TypeScript
```

Comment:
Our SDK for MediaComposition is meant for Contoso server not client. IMHO, only server languages would make sense. So probably we wouldn't want Python. Ruby and NodeJS make sense but we'd more than likely want Java. Calling Server SDK only provide C# support. We may want to be progressive meaning start with C# and add other languages as we need.

Reply:
You're right. I left only the C# readme here.

I see some of our partner teams (such as Calling Server) don't have a `Java readme` even though they have a Java SDK so I didn't include in ours either.